### ISSUE TITLE
Automated reward distribution

### ISSUE THREAD

**Issue by totalvm** [CityOfZion/developer-portal#6]

**Comment by canesin** [comment]
It should be a dApp that is a interface for a mult-sig .. so we vote in a reward structure and on agreement submit to the SC a distribution matrix, we then have to sign the approval ...

**Comment by totalvm** [comment]
@canesin 
Yes that was my plan as well in the end. I will have to maybe get someone to help set up the contract for me as doing it myself will take too much time.
